idea-plugin/src/test/java/com/palantir/gradle/versions/intellij/VersionPropsCodeInsightTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.gradle.versions.intellij;
+
+import com.intellij.codeInsight.completion.CompletionType;
+import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class VersionPropsCodeInsightTest extends LightJavaCodeInsightFixtureTestCase {
+    @Override
+    protected final String getTestDataPath() {
+        return "src/test/testData";
+    }
+
+    @Test
+    public void test_version_completion() throws Exception {
+        setUp();
+        myFixture.configureByFiles("VersionTestData.props");
```

Comment:
are you able to use `configureByText` instead? If it's not a large amount of text, it makes it easier to process the test as you don't have to jump around to find the underlying file. Which in this case is not as straightforward as you actually have to plumb through IntelliJ's test helpers

Reply:
switched to configure by text thanks!